xonsh/webconfig/file_writes.py

```diff
@@ -39,7 +40,7 @@ def config_to_xonsh(
     for func in RENDERERS:
         lines.extend(func(config))
     lines.append(suffix)
-    return "\n".join(lines)
+    return re.sub(r"\\r", "", "\n".join(lines))
```

Comment:
Does a `replace` on the joined string work here?  Not a blocker, just curious

Reply:
sorry for the late reply. when testing i found it does work and the prompt in the config file is stripped of the carriage return.